uninstall.php

```diff
@@ -0,0 +1,173 @@
+<?php
+/**
+ * Distributor uninstall script.
+ *
+ * @package distributor
+ * @since   x.x.x
+ */
+
+// phpcs:disable
+
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	die;
+}
+
+/*
+ * Only remove ALL data if DT_REMOVE_ALL_DATA constant is set to true in user's
+ * wp-config.php. This is to prevent data loss when deleting the plugin from the backend
+ * and to ensure only the site owner can perform this action.
+ */
+if ( defined( 'DT_REMOVE_ALL_DATA' ) && true === DT_REMOVE_ALL_DATA ) {
+	global $wpdb;
+
+	/**
+	 * Function to delete all relevant data from the site.
+	 */
+	function dt_delete_data() {
+		global $wpdb;
+
+		// Delete post meta and posts of type 'dt_subscription'.
+		$subscription_post_ids = $wpdb->get_col(
+			$wpdb->prepare(
+				"SELECT ID FROM $wpdb->posts WHERE post_type = %s",
+				'dt_subscription'
+			)
+		);
+
+		if ( ! empty( $subscription_post_ids ) ) {
+			$ids_string = implode( ',', array_map( 'intval', $subscription_post_ids ) );
+
+			// Delete subscription meta.
+			$wpdb->query(
+				$wpdb->prepare(
+					"DELETE FROM $wpdb->postmeta WHERE post_id IN ($ids_string)"
+				)
```

Comment:
```suggestion
				"DELETE FROM $wpdb->postmeta WHERE post_id IN ($ids_string)"
```

No need to use `prepare` without placeholders. :1234: Applies elsewhere in this PR.

That said, I think I'd prefer to use the `implode...count` method to generate the placeholders and use a prepare statement just to be safe. See the code in `wp_prime_options_cache()` https://github.com/WordPress/WordPress/blob/4d0a623d7a01ca03aca529bfb35d7bca7c35ba48/wp-includes/option.php#L261